components/gardener/extensions/deployment.yaml

```diff
@@ -61,7 +61,8 @@ extension_controllers_values:
     createCRDs: false
     fullnameOverride: seed-dns-controller-manager
     image:
-      tag: 0.3.3
+      repo: (( .landscape.versions.dns_controller.image_repo ))
+      tag: (( .landscape.versions.dns_controller.image_tag ))
```

Comment:
I'm not completely sure we should do this. We have two dns-controller-managers here - the one this project uses and the one used by the gardener. This change will set both to the same image and version. Maybe it would be a better idea to create two different entries in the acre.yaml (which still default to the same version) and reference the second one here, so that it is possible to have different images or versions for garden-setup and gardener. 

What do you think?